homeassistant/components/manual_mqtt/alarm_control_panel.py

```diff
@@ -45,16 +47,24 @@
 CONF_PAYLOAD_ARM_HOME = "payload_arm_home"
 CONF_PAYLOAD_ARM_AWAY = "payload_arm_away"
 CONF_PAYLOAD_ARM_NIGHT = "payload_arm_night"
+CONF_PAYLOAD_INVALID = "payload_invalid"
+CONF_CONFIG_TOPIC = "config_topic"
+CONF_STATUS_TOPIC = "status_topic"
 
 DEFAULT_ALARM_NAME = "HA Alarm"
-DEFAULT_DELAY_TIME = datetime.timedelta(seconds=0)
-DEFAULT_PENDING_TIME = datetime.timedelta(seconds=60)
+DEFAULT_DELAY_TIME = datetime.timedelta(seconds=60)
```

Comment:
Yeah, it is better to have a bit of  delay here.  Wondering what about the difference between the delay to arm home vs delay arm away.   Arm home usually has a zero delay, arm away usually has 60 seconds.

Reply:
Right now there is no difference between the arming time for arm_home, arm_away, or arm_night.  If you want to declare those, you need to do it in the config.

Reply:
Yeah I think is how it currently works, just double checking.  I usually set an alarm home delay for 0 and 60 seconds for away mode.   The delay for Triggered is misleading as for some reason it was originally designed to be how long to have trigger state, but most alarms are triggered until the code is entered manually.

Reply:
Yes, I do too (set arm_home delay to 0, arm_away delay to 20).  This is in the configuration.
I think the delay for triggered is more about how long the siren would sound even without a code.  In some places it may be required by law that the siren turn off after, say, 10 minutes.  This code was there before and I didn't change that logic.

Reply:
Man, all the things I did not know about alarms.  Thanks!